lasagne/layers/special.py

```diff
@@ -702,8 +736,8 @@ def _transform_thin_thin_plate_spline(dest_offsets, input, source_points,
     # dimshuffle input to  (bs, height, width, channels)
     input_dim = input.dimshuffle(0, 2, 3, 1)
     input_transformed = _interpolate(
-            input_dim, x_transformed, y_transformed,
-            out_height, out_width)
+        input_dim, x_transformed, y_transformed,
+        out_height, out_width)
```

Comment:
I'd prefer double indent as before (make sure to use the same pep8 version as in `requirements-dev.txt`).